performance-tests/script.ts

```diff
@@ -0,0 +1,104 @@
+import { Output, wrap } from "../src/implementations/wrap";
+import { performance } from "perf_hooks";
+import { existsSync, mkdirSync, rmdirSync, promises } from "fs";
+import { SchemaId } from "@govtechsg/open-attestation";
+import { join } from "path";
+
+const DEFAULT_NUMBER_OF_FILE = 2;
+const DEFAULT_ITERATION = 1;
+const DEFAULT_FILE_PATH = join(__dirname, "/unwrapped_doc.json");
```

Comment:
the purpose of join is to NOT have to specify the path separator, which is different on Linux and windows